docs/source/index.md

```diff
@@ -11,76 +11,87 @@ StreamingDataset is compatible with any data type, including **images, text, vid
 from torch.utils.data import DataLoader
 from streaming import StreamingDataset
 
-dataloader = DataLoader(dataset=StreamingDataset(remote='s3://...'))
+dataloader = DataLoader(dataset=StreamingDataset(remote='s3://...', batch_size=1))
 ```
 
-
-
+## **💾** Installation
+1. Set up your Python development environment.
+2. Install Streaming with `pip`:
+```
+pip install mosaicml-streaming
+```
+3. Verify the installation with:
+```
+python -c "import streaming; print(streaming.__version__)"
+```
+4. Jump to our [Quick Start](getting_started/quick_start.md) and [Main Concepts](getting_started/main_concepts.md) guides.
 
 ## **🔑** Key Features
```

Comment:
Should we also add `tensor or sequence parallelism`?

Reply:
done